tests/bug-detectors/package.json

```diff
@@ -3,8 +3,8 @@
 	"version": "1.0.0",
 	"description": "Tests for Jazzer's bug detectors",
 	"scripts": {
-		"fuzz": "jest",
-		"test": "jest"
+		"test": "jest",
+		"fuzz": "jest"
```

Comment:
Why do both point to the same command?

Reply:
For unexplainable reasons :laughing: Deleted "test".